rosdep/python.yaml

```diff
@@ -207,7 +207,7 @@ jupyter-nbconvert:
   gentoo: [dev-python/nbconvert]
   ubuntu:
     '*': [jupyter-nbconvert]
-    xenial: null
+    xenial:
```

Comment:
This change looks unrelated to your pull request. Please remove this part of the diff.